bin/generate-docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Check from other scripts how to name the first line. Also rename `generate-docs.sh` to `generate-docs`

Reply:
I see other scripts having the exact same line, can you tell in more detail what you mean by naming the first line?

Reply:
They are `#!/bin/sh` / `#!/bin/bash` apparently. Only remove the `.sh` ending from the name.